docs/developers-guide/devenv.md

```diff
@@ -174,6 +174,11 @@ You could also pass a full conection string in as the `mb.db.connection.uri`:
 "-Dmb.db.connection.uri=postgres://<user>:<password>@localhost:5432/<dbname>"
 ```
 
+The other way to override config variables like that is to use `.lein-env` file in the top level of repo:
```

Comment:
I don't think we should encourage ppl to use `.lein-env`, we actually plan to remove support for this.

see https://github.com/metabase/metabase/blob/invalid-params-reporting/src/metabase/config.clj#L152

Reply:
ehhh, but that's a good way to override some configuration without editing `deps.edn`. Overlaying multiple `deps.edn` is a bit of a pain honestly. :)

I agree `.lein-env` is bad name, but `environ` design prevents passing another file name. I obviously could point to a `resources/.boot-env`, but it's essentially the same.

Reply:
I see your point.

I'm not against this strongly but I feel like we should guide ppl toward using a mechanism we explicitly support.
Another reason we should use env is it works for both starting a repl and running a jar.

Reply:
Yeah I'm certainly inclined to use this feature as it makes things a bit more ergonomic for me, but IDK if putting it in documentation is the right thing to do...

As long as we don't drop `environ` from deps, jar is going to be reading `.lein-env` from a working directory, I guess. :-)